package.json

```diff
@@ -28,6 +28,7 @@
     "gatsby-transformer-remark": "^6.11.0",
     "gatsby-transformer-sharp": "^5.11.0",
     "react": "^18.2.0",
+    "react-copy-to-clipboard": "^5.1.0",
```

Comment:
We have already native API for this, is this really required

https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API

Reply:
@aadilmughal786 ,

- I had no idea there was a native API for Clipboard when I was looking into this feature, and when I explored this dependency, I believed the simplicity of usage was great for me, so I didn't dig more.
- After you pointed out that there is a native API for the clipboard I saw the difference between both I found out that there is not a lot react-copy-to-clipborad does. simply, dependency is not required (except for adding a dependency for my project). Now switching to native API

Thanks for pointing that out.